src/test/resources/pages/adminActivityLogPage.html

```diff
@@ -261,9 +256,11 @@
             <ul class="nav navbar-nav pull-right">
                 <li>                
                     <a href="/logout.jsp" class="nav logout">
-                        <span class="glyphicon glyphicon-user"></span>
-                        Logout  (<span title="" data-placement="bottom" data-toggle="tooltip" class="text-info" data-original-title="defaultAdmin"> defaultAdmin
-                        </span>) 
+                        <span class="glyphicon glyphicon-user"></span> Logout
+                        
+                        (<span title="" data-placement="bottom" data-toggle="tooltip" class="text-info" data-original-title="defaultAdmin">
+                                defaultAdmin
```

Comment:
I'm aware of this. All the other `Admin*UiTest` are guilty of this as well. Putting a wild card works, but may not be the only solution. Let's fix them all together at another issue.